p2p/conn/connection.go

```diff
@@ -664,6 +704,31 @@ FOR_LOOP:
 	}
 }
 
+func (c *MConnection) recvMsgRoutine() {
+	defer c._recover()
+
+	for {
+		connMsg, ok := <-c.msgRecvQueue
+
+		if !ok {
+			break
```

Comment:
When is this `break` expected to be encountered? If it's an error condition, something noisier is warranted. But if it's just routine shutdown from `recvRoutine` closing the channel, a comment to that effect would be nice.

Reply:
routine shutdown